background/background.js

```diff
@@ -289,32 +289,53 @@ const LivemarkUpdater = {
 
 const ContextMenu = {
   async init() {
-    const createReloadItem = async () => {
+    const getUnreadChildren = async (id) => {
+      const p = await Settings.getReadPrefix();
+      const feed = [...await browser.bookmarks.getChildren(id)]
+        .filter((e,i,arr) => !arr.slice(i).some(e => e.type==="separator"))
+        .filter(e => !PrefixUtils.hasPrefix(p, e.title));
```

Comment:
Doesn't that mean it won't work for people who have prefixes disabled?

Reply:
Yes, I'll correct this